dspace/config/emails/bte_batch_import_success

```diff
@@ -0,0 +1,13 @@
+## Email sent to DSpace users when they successfully batch import items via BTE.
```

Comment:
This email appears to be used [here](https://github.com/TAMULib/DSpace/blob/tamu-dspace-7_x/dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportServiceImpl.java#L2236).